pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -485,6 +485,9 @@ message TableReplicationMetadata {
   optional string destination_parent_database_name = 2 [(gogoproto.nullable) = false];
   optional string destination_parent_schema_name = 3 [(gogoproto.nullable) = false];
   optional string destination_table_name = 4 [(gogoproto.nullable) = false];
+  // DestinationFunctionOid, if non-zoro, is the name of the
```

Comment:
```suggestion
  // DestinationFunctionOid, if non-zero
```
Should name be OID here in this comment since that's what it is stored as?

Reply:
I kept going back and forth. The type is `oid.Oid`. But I suppose all of our functions use OID.